src/routes/Dashboard/Dashboard.svelte

```diff
@@ -39,6 +40,8 @@
   let errorType;
   let loading = true;
 
+  let eventsUrl = `https://notehub.io/project/${APP_UID}/events?queryMode=devices&queryDevices=${deviceUID}`;
```

Comment:
This same URL is also in `Settings.svelte` so it might be worth moving to some utility.

Reply:
Your second comment allowed me to remove this duplicate code :)